ext/js/display/display-anki.js

```diff
@@ -394,6 +424,7 @@ export class DisplayAnki {
                     this._setupTagsIndicator(i, noteInfos);
                 }
             }
+
```

Comment:
Don't modify whitespace outside the scope of the change

Reply:
done

Reply:
actually, looks like I made a mistake and forgot to remove the whitespace here